MAINT-25739: Session TIMEOUT Problem, No popup is appeared when the session expired

AzmiTouil opened the pull request:
Add a popup when session expires to tell user to backup your editor content

AzmiTouil committed:
MAINT-25739: Session TIMEOUT Problem, No popup is appeared when the session expired

ahamdi reviewed (changes requested):
We can not rely on Javascript to count when a session is expired. Check PortalHttpRequest.js to see how to handle the session expiration for Ajax requests

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

Rdidmohamedelhadi committed:
QWIN-36683 : Getting Started Improvements (add button close) (#418)

rdenarie committed:
Task-36806 : When importing csv user file with a new line at the end, we cannot import a new file after (#419)

Before this commit, when reading the csv file, if a line is empty, we don't read a new line, so we are stuck in infinite loop, preventing the executor to start new task
This commit add a readLine instruction when the read line is empty, so that we can continue in the file and reach the end.

Rdidmohamedelhadi committed:
QWIN-36683 : Getting Started Improvements

ahamdi committed:
35824 : Add lastUpatedTime property to space (#375)

A new property was added to the space entity to store the last updated time. This property will be used to add an ETAG to the Rest response when requesting a space.
